docker/web/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# This is the web only image of cBioPortal. Useful when you would like to run a
+# small image without all data import/validate related scripts and
+# dependencies. It includes the ShenandoahGC
+# (https://wiki.openjdk.java.net/display/shenandoah/Main), an experimental
+# garbage collector which in our experience works better in a cloud environment
+# where you have multiple containers running on a single node. It gives unused
+# memory back to the system allowing other containers to utilize it. Enable
+# this GC with JAVA_OPTS=-XX:+UnlockExperimentalVMOptions -XX:+UseShenandoahGC
+FROM maven:3.5.4 as build
+COPY $PWD /cbioportal
+WORKDIR /cbioportal
+RUN mvn -DskipTests clean install
+
+FROM shipilev/openjdk-shenandoah:8
+COPY --from=build /cbioportal/portal/target/cbioportal*.war /app.war
+COPY --from=build /cbioportal/portal/target/dependency/webapp-runner.jar /webapp-runner.jar
+CMD /usr/bin/java ${JAVA_OPTS} -jar /webapp-runner.jar ${WEBAPP_OPTS} /app.war
```

Comment:
can we  have a Dockerfile for just data loading? for instance have Dockerfiles: 

- web+data
- web only 
- data loading only

Reply:
@khzhu it would be nice to have a data loading only image, but note that it's not that useful b/c of how connected the data import and core module still is. You still need Java+Python. The only difference with the `web+data` one would be that there is no war file and `webapp-runner` included. These are however not that big. So it seems to me like this would only really be useful once we move the core module out and completely separate the data import code from the web code

Reply:
@inodb @fedde-s , I got data loading only image worked out. The size of image is not that much reduced (basically only excluding two jar files), but the new image is more user friendly, since all a user needs to do is to mount a volume to a docker container. Something like below:
```
docker run -it -v /Users/kzhu/Downloads/breast_msk_2018:/study kzhupm/cbio-loader
```
I agree with you both and will not complicate things here. I will open a new PR for new data loader Docker file at some point in the near future. I will also include two optimized docker files for existing web and web-and-data (Elena from Azure team helped with the optimization, the web image now is only 282MB).

Reply:
@khzhu that's great! Thanks!

Reply:
Sounds like a great idea, even if it's just a different default command; I never thought of that!